core/pkg/evaluator/json_model.go

```diff
@@ -10,14 +10,9 @@ type Evaluators struct {
 	Evaluators map[string]json.RawMessage `json:"$evaluators"`
 }
 
-type Metadata struct {
-	FlagSetID      string `json:"flagSetId"`
-	FlagSetVersion string `json:"flagSetVersion"`
-}
-
 type ConfigWithMetadata struct {
-	Flags    map[string]model.Flag `json:"flags"`
-	MetaData Metadata              `json:"metadata"`
+	Flags    map[string]model.Flag  `json:"flags"`
+	MetaData map[string]interface{} `json:"metadata"`
```

Comment:
```suggestion
	Metadata map[string]interface{} `json:"metadata"`
```

This will also need to change everywhere it's being used.

Reply:
Done